modules/lobbies/utils/rivet/geo_coord.ts

```diff
@@ -0,0 +1,41 @@
+import { ScriptContext } from "../../module.gen.ts";
+import { RegionGeoCoords, EMPTY_GEO_COORDS } from "../region.ts";
+
+type TraceEntry = ScriptContext["trace"]["entries"][0];
+type TraceEntryTypeHttpRequest = Extract<TraceEntry["type"], { httpRequest: any }>["httpRequest"];
+export const getRequestGeoCoords = (
+    ctx: ScriptContext
+): RegionGeoCoords => {
+    // If they aren't real servers, we're probably not in managed
+    // So we just return
+    if (!("server" in ctx.config.lobbies.backend)) return EMPTY_GEO_COORDS;
+
+    // Check if env flag is set for managed
+    // TODO(Nathan): Set this flag to 1 for managed
+    if (ctx.environment.get("MANAGED") !== "1") return EMPTY_GEO_COORDS;
```

Comment:
`RIVET_BACKEND_RUNTIME=cloudflare_workers_platforms`. this matches the `rivet backend build --runtime` flag.